crates/revm/revm-primitives/src/env.rs

```diff
@@ -38,7 +38,6 @@ pub fn fill_cfg_env(
 
     cfg_env.chain_id = U256::from(chain_spec.chain().id());
     cfg_env.spec_id = spec_id;
-    cfg_env.perf_all_precompiles_have_balance = false;
```

Comment:
should we change this?

Reply:
Option was removed from revm, we handle precompiles differently